src/opuntia.hrl

```diff
@@ -0,0 +1,10 @@
+-ifndef(OPUNTIA).
```

Comment:
I wouldn't because then users of this library would do
```erl
-include_lib("opuntia/include/opuntia.hrl")
```
And I don't want them to look at the internals of the record 🤔

Reply:
Oh... ok. I guess it's justified then.